spec/presenters/summary/sections/overview_section_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe Summary::Sections::Overview do
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) do
+    instance_double(
+      CrimeApplication,
+      to_param: '12345',
+      status: status,
+      reference: 'LAA-1',
```

Comment:
We can use a numeric value (given reference/USN will be numeric) here, and then below in the expectation we check it has been casted to a string, but this is a minor thing.